lib/player_x.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'game'
+require_relative 'board'
+
+class PlayerX
+
+  attr_reader :name, :board
+
+  def initialize(board)
+    @name = 'PlayerX'
+    @board = board
+  end
+
+  def move(a,b)
+    raise 'This position is already taken' unless @board.grid[a][b] == '-'
```

Comment:
Encapsulation: Should the Player class know how to check if a board position is available?